applications/ws-fund-ner/config.json

```
@@ -0,0 +1,20 @@
+{
+	"environnement": {
+		"EZS_TITLE": "ws-funder-ner",
+		"EZS_DESCRIPTION": "Web service detection de financeur",
+		"EZS_METRICS": true,
+		"EZS_CONCURRENCY": 4,
+		"EZS_NSHARDS": 32,
+		"EZS_CACHE": true,
+		"NODE_OPTIONS": "--max_old_space_size=1024",
+		"NODE_ENV": "production",
+		"PIP_NO_DEPENDENCIES": 0,
+		"PIP_DISABLE_PIP_VERSION_CHECK": 1,
+		"PIP_NO_CACHE_DIR": 0
+	},
+	"packages": [
+		"@ezs/core@2.4.3",
+		"@ezs/spawn@1.2.30",
+		"@ezs/basics@1.23.3"
```

These are old versions.

```suggestion
		"@ezs/core@3.3.0",
		"@ezs/spawn@1.4.5",
		"@ezs/basics@2.5.0"
```